src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -196,30 +266,74 @@ const TableToolbar = ({
           disabled={isDisabled}
         />
       ) : null}
-      {customToolbarContent || null}
-      {totalFilters > 0 ? (
-        <Button kind="secondary" onClick={onClearAllFilters}>
-          {i18n.clearAllFilters}
-        </Button>
-      ) : null}
-      {onDownloadCSV ? (
-        <ToolbarSVGWrapper onClick={onDownloadCSV}>
-          <Download16 description={i18n.downloadIconDescription} />
-        </ToolbarSVGWrapper>
-      ) : null}
-      {hasColumnSelection ? (
-        <ToolbarSVGWrapper onClick={onToggleColumnSelection}>
-          <IconColumnSelector description={i18n.columnSelectionButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-      {hasFilter ? (
-        <ToolbarSVGWrapper onClick={onToggleFilter}>
-          <IconFilter description={i18n.filterButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-    </StyledTableToolbarContent>
-  </StyledCarbonTableToolbar>
-);
+      {activeEditBar ? (
+        <StyledTableToolbarContent>
+          <Button className="buttons" kind="ghost" onClick={onCancelEditAction}>
+            {i18n.batchCancel}
+          </Button>
+          <Button className="buttons" onClick={onSaveEditAction}>
+            {i18n.batchSave}
+          </Button>
+        </StyledTableToolbarContent>
+      ) : showAlert ? (
+        <ToastNotification
+          className="toast"
+          caption=""
+          hideCloseButton={false}
+          iconDescription="undo changes and close"
+          kind="success"
+          lowContrast
+          notificationType="toast"
+          onCloseButtonClick={onAlertClose}
+          role="alert"
+          subtitle="Click to undo changes"
```

Comment:
If you are going to leave the Toast in the Toolbar, then, you'll need to add i18n string for all the text that you are using.  Personally, I don't think the "table" or "toolbar" should be firing a "toast" message.  I think the story can simulate this simply by hooking into the onSave and then showing it's own Toast from the story.